.github/workflows/sentry_rails_test.yml

```diff
@@ -43,6 +43,9 @@ jobs:
           - { ruby_version: "2.7", rails_version: 5.2.0 }
           - { ruby_version: "2.7", rails_version: 6.0.0 }
           - { ruby_version: "2.7", rails_version: 6.1.0 }
+          - { ruby_version: "3.1", rails_version: 7.2.0 }
+          - { ruby_version: "3.2", rails_version: 7.2.0 }
+          - { ruby_version: "3.3", rails_version: 7.2.0 }
```

Comment:
I know it's ultimately a business decision, but IMO we should drop support or at least stop running CI against Ruby 2.4, 2.5 and Rails 5.0 and 5.1.
Otherwise, with the release of Rails 8.0 in the near future, we'll be running so many jobs on CI for each PR.

Reply:
I will prepare a major soon and remove, just need some time since I'm busy with another larger project